Update Prevalue referenes to Options (#6359)
umbraco/UmbracoDocs: 14/umbraco-cms/fundamentals/data/data-types/README.md

## Creating a new Data Type

Follow these steps to create a new Dropdown Data Type:

1. Go to the **Settings** section within the backoffice.
2. Select the **+** icon to the right of the **Data Types** folder.
3. Choose **New Data Type...**.
4. Name the Data Type.
5. Click on **Select a property editor**.

> 6. Find a click on the **Dropdown** editor.

Reviewer: ```suggestion
6. Find and click on the **Dropdown** editor.
```

7. Click **Select**.
8. Choose whether to enable multiple selections.
9. Add **options**.
10. **Save** the Data Type once you have added the required configuration.

![Dropdown List](images/dropdown-data-type-sample.png)

{% hint style="info" %}
**Data Type configuration**

> **Property Editor** This is where you pick the Property Editor UI that the Data Type will be referencing. By default Umbraco ships with wide selection to choose between. Learn more about each of them in the [Default Data Types](./default-data-types.md) article.

Reviewer: ```suggestion
**Property Editor** This is where you pick the Property Editor UI that the Data Type will be referencing. By default, Umbraco ships with a wide selection to choose from. Learn more about each of them in the [Default Data Types](./default-data-types.md) article.
```

> In the **Settings** box below is where the configuration options specific to the chosen Property Editor UI will be available. Some Property Editors has many configuration options while some only has a few.

Reviewer: ```suggestion
In the **Settings** box below, the configuration options specific to the chosen Property Editor UI will be available. Some Property Editors have many configuration options while some only have a few.
```
